oshi-core/src/site/site.xml

```diff
@@ -25,28 +25,28 @@
 
 -->
 <project xmlns="http://maven.apache.org/DECORATION/1.8.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="${project.name}"
-	xsi:schemaLocation="http://maven.apache.org/DECORATION/1.8.0 http://maven.apache.org/xsd/decoration-1.8.0.xsd">
-	<skin>
-		<groupId>org.apache.maven.skins</groupId>
-		<artifactId>maven-fluido-skin</artifactId>
-		<version>1.7</version>
```

Comment:
if you define this version as a property in the pom eg. `<maven-fluido-skin.version>...</maven-fluido-skin.version>` and add it as a dependency of the `maven-site-plugin`  like

```xml
<plugin>
    <artifactId>maven-site-plugin</artifactId>
    <version>3.10.0</version>
    <dependencies>
        <dependency>
            <groupId>org.apache.maven.skins</groupId>
            <artifactId>maven-fluido-skin</artifactId>
            <version>${maven-fluido-skin.version}</version>
        </dependency>

   ...
```

you can use `${maven-fluido-skin.version}`  here and dependabot will keep it up-to-date.


example:
https://github.com/GeoDienstenCentrum/schemaspy-maven-plugin/blob/a6c06701115f98f2b7e4903a577908899587076a/pom.xml#L296-L311
and
https://github.com/GeoDienstenCentrum/schemaspy-maven-plugin/blob/a6c06701115f98f2b7e4903a577908899587076a/src/site/site.xml#L1-L12

Reply:
@mprins Nice!  Thank you that is much cleaner.  Will adjust to that.

Reply:
We were already tracking on site so versions plugin would note the update.  Dependabot was doing that bit as well as versions plugin is pretty useless in that regard now.  All the site pages have been updated.  I do believe technically speaking all sites inherit from each other so it is only really needed in the first but for simplicity given these are all the same, no reason to be different :)